os/Kconfig

```diff
@@ -406,6 +406,599 @@ config ARCH_STDARG_H
 
 endmenu # Customize Header Files
 
+menu "Debug Options"
```

Comment:
I just added only audio debug here. please check it.

Reply:
Oh man.. dont revert our changes. This block was removed. Check Kconfig.debug

Reply:
hmm I did it based on latest code of internal git.
What is correct process?????
I thought internal -> public is correct process.
So many things are different, we need your member's review more.

Reply:
fixed.